File: Database/DataAccess/models/Preset.cs
using System;
using System.Collections.Generic;

namespace DataAccess.models;

public partial class Preset
{
    public int Id { get; set; }

    public string Name { get; set; } = null!;

    public int MinHumidity { get; set; }

    public int MaxHumidity { get; set; }

    public int MinCo2 { get; set; }

    public int MaxCo2 { get; set; }

    public int MinTemperature { get; set; }

    public int MaxTemperature { get; set; }

    public int? DeviceId { get; set; }

    public virtual Iotdevice? Device { get; set; }
}

File: WebSockets/WebSocketsGateway/src/main/java/DTO/Preset.java
package DTO;


import lombok.Data;
import org.springframework.stereotype.Component;

@Component
@Data
public class Preset {
    private int Id;
    private String Name;
    private float MinHumidity;
    private float MaxHumidity;
    private float MinTemperature;
    private float MaxTemperature;
    private float MinCo2;
    private float MaxCo2;
    private String DeviceId;

}
